app/models/visit.rb

```diff
@@ -76,4 +76,8 @@ def banned_visitors
   def unlisted_visitors
     visitors.unlisted
   end
+
+  def unavailable?
```

Comment:
I don't think `unavailable?` is the right word here: it implies something different about the visit. Perhaps `awaiting_processing?` might be closer.